lib/bundler/resolver.rb

```diff
@@ -208,6 +208,7 @@ def sort_dependencies(dependencies, activated, conflicts)
         [
           @base_dg.vertex_named(name) ? 0 : 1,
           activated.vertex_named(name).payload ? 0 : 1,
+          activated.vertex_named(name).root? ? 0 : 1,
```

Comment:
let's extract `activated.vertex_named(name)` into a local variable, since it's used multiple times